src/resolution_functions/models/vision_paper.py

```diff
@@ -148,4 +152,22 @@ def __call__(self, frequencies: Float[np.ndarray, 'frequencies']) -> Float[np.nd
         sigma *= 2 * e1
         sigma -= self.final_term
 
-        return sigma
+        return {'sigma': sigma}
```

Comment:
Is it still "sigma" or something else? I notice the __call__ docstring refers to FWHM.